macros/CSP.ejs

```diff
@@ -1,20 +1,20 @@
 <%
 
-var lang = env.locale;
-var directive = $0;
-var str = $1 || $0;
-var rtlLocales = ['ar', 'he', 'fa'];
-var localStrings = string.deserialize(template("L10n:Common"));
-var URL = "/" + lang + '/docs/Web/HTTP/Headers/Content-Security-Policy/' + directive;
-var anch = '';
+let lang = env.locale;
+let directive = $0;
+let str = $1 || $0;
+let rtlLocales = ['ar', 'he', 'fa'];
+let localStrings = string.deserialize(await template("L10n:Common"));
+let URL = "/" + lang + '/docs/Web/HTTP/Headers/Content-Security-Policy/' + directive;
```

Comment:
@davidflanagan As we are using some ES6 goodies, should we not consider using template literals as well? Might it cause problems with the way the string is built?

Reply:
We're running in node 10, so all of ES6 is up for grabs if macro authors want to use it. 

For this patch, though, I'm trying to touch the macros as little as possible, so I don't want to add any more changes.